phase/gather_k0s_facts.go

```diff
@@ -83,6 +85,50 @@ func (p *GatherK0sFacts) Run() error {
 		return err
 	}
 
+	if p.leader.Metadata.K0sRunningVersion != nil {
+		if err := p.listEtcdMembers(p.leader); err != nil {
```

Comment:
What about clusters that are not running internal etcd, i.e,. running external etcd or kine instead?

Reply:
Yes I was thinking that too, needs some extra conditions.